go.mod

```diff
@@ -48,12 +62,12 @@ require (
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
 	github.com/gomodule/redigo v1.8.9 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
-	github.com/klauspost/compress v1.15.9 // indirect
+	github.com/klauspost/compress v1.15.15 // indirect
 	github.com/mattn/go-runewidth v0.0.10 // indirect
 	github.com/olekukonko/tablewriter v0.0.5 // indirect
-	github.com/pierrec/lz4/v4 v4.1.15 // indirect
+	github.com/pierrec/lz4/v4 v4.1.17 // indirect
 	github.com/segmentio/encoding v0.3.5 // indirect
 	golang.org/x/sys v0.5.0 // indirect
 )
 
-replace github.com/segmentio/parquet-go v0.0.0-20230427215636-d483faba23a5 => github.com/chelseajonesr/parquet-go v0.0.3
+replace github.com/apache/arrow/go/v13 => github.com/chelseajonesr/arrow/go/v13 v13.0.0-20230711200800-c7890b0a2007
```

Comment:
This redirection is for a minor patch to allow Arrow to skip serializing/deserializing records to Arrow when the tag is "-", similarly to JSON.  It's been accepted into the Arrow 14 branch as of today so I'll likely remove this redirection in the optimization branch that is in progress,